tasks/unit_tests/omnibus_tests.py

```diff
@@ -103,16 +103,16 @@ def test_successful_cache_hit(self):
         # shouldn't have been bundled and uploaded
         commands = _run_calls_to_string(self.mock_ctx.run.mock_calls)
         lines = [
-            'git -C omnibus-git-cache/opt/datadog-agent bundle create /tmp/omnibus-git-cache-bundle --tags',
-            r'aws s3 cp (\S* )?/tmp/omnibus-git-cache-bundle s3://omnibus-cache/builds/\w+/slug',
+            r'git -C omnibus-git-cache/opt/datadog-agent bundle create /\S+/omnibus-git-cache-bundle --tags',
+            r'aws s3 cp (\S* )?/\S+/omnibus-git-cache-bundle s3://omnibus-cache/builds/\w+/slug',
         ]
         for line in lines:
             self.assertIsNone(re.search(line, commands))
 
     def test_cache_miss(self):
         self.mock_ctx.set_result_for(
             'run',
-            re.compile(r'aws s3 cp (\S* )?s3://omnibus-cache/builds/\S* /tmp/omnibus-git-cache-bundle'),
+            re.compile(r'aws s3 cp (\S* )?s3://omnibus-cache/builds/\S* /\S+/\S+/omnibus-git-cache-bundle'),
```

Comment:
Is it expected to have a double `\S+` here and not on other places?

Reply:
Indeed that's not needed, I'll remove the duplicate